README.md

```diff
@@ -92,6 +92,25 @@ Like any open-source software, Arlo welcomes suggested changes in the form of pu
 
 ### Setting up the dev environment
 
+#### Linux
+
+We recommend Ubuntu 18.0.4.
+
+* Install Node10. See https://joshtronic.com/2018/05/08/how-to-install-nodejs-10-on-ubuntu-1804-lts/
+* `sudo make deps`
```

Comment:
These instructions don't make use of the `dev-environment` task. Also, it implies that you need `sudo` to run `deps`, which `dev-environment` doesn't do since `deps` is run as a dependency.